content/applications/inventory_and_mrp/inventory/product_management/product_tracking/serial_numbers.rst

```diff
@@ -93,192 +132,209 @@ product that this newly-created serial number was just assigned to.
-from a purchase order. **Existing** serial numbers cannot be used. For sales orders, the opposite is
-true: new serial numbers cannot be created on the delivery order, only existing serial numbers can
-be used.
-
-To change the ability to use new (or existing) serial numbers on any operation type, go to
-:menuselection:`Inventory app --> Configuration --> Operations Types`, and select the desired
-:guilabel:`Operation Type`.
-
-For the :guilabel:`Receipts` operation type, found on the :guilabel:`Operations Types` page, the
-:guilabel:`Use Existing Lots/Serial Numbers` option can be enabled, by selecting
-:guilabel:`Receipts` from the :guilabel:`Operations Types` page, clicking :guilabel:`Edit`, and then
-clicking the checkbox beside the :guilabel:`Use Existing Lots/Serial Numbers` option (in the
-:guilabel:`Traceability` section). Lastly, click the :guilabel:`Save` button to save the changes.
-
-For the :guilabel:`Delivery Orders` operation type, located on the :guilabel:`Operations Types`
-page, the :guilabel:`Create New Lots/Serial Numbers` option can be enabled, by selecting
-:guilabel:`Delivery Orders` from the :guilabel:`Operations Types` page, clicking :guilabel:`Edit`,
-and clicking the checkbox beside the :guilabel:`Create New Lots/Serial Numbers` option (in the
-:guilabel:`Traceability` section). Be sure to click :guilabel:`Save` to save changes.
-
-.. image:: serial_numbers/serial-numbers-operations-types.png
+In the :guilabel:`Lot/Serial Number` column, manually type (or select from the dropdown)
+the desired serial numbers for each individual product.
+
+.. image:: serial_numbers/moves-button.png
    :align: center
-   :alt: Enabled traceability setting in operations type form.
+   :alt: Show the detailed Moves page.
+
+When finished, click the receipt/delivery order's breadcrumb trail, and the assigned serial numbers
+will be automatically saved.
+
+Traceability & Reporting
+========================
 
-Serial number traceability
-==========================
+Manufacturers and companies can refer to the :guilabel:`Lots/Serial Numbers dashboard` and
+traceability reports to see the entire lifecycle of a product: when and where it originated,
+where it was stored, and who it was shipped to.
 
-Manufacturers and companies can refer to the traceability reports to see the entire lifecycle of a
-product: where it came from (and when), where it was stored, and who it went to.
+Lots/Serial Numbers dashboard
+-----------------------------
 
 To see the full traceability of a product, or group by serial numbers, go to
 :menuselection:`Inventory app --> Products --> Lots/Serial Numbers`. Doing so reveals the
 :guilabel:`Lots/Serial Numbers` dashboard.
 
-From here, products with serial numbers assigned to them will be listed by default, and can be
-expanded to show what serial numbers have been specifically assigned to them.
+Reporting
+---------
+
+From here, products with serial numbers assigned to them will be listed by default. Click the
```

Comment:
We tend to avoid starting paragraphs with "here" or "there". We do it in the middle of a sentence or in eLearning tutorials (:

How about moving the "Doing so reveals the
:guilabel:`Lots/Serial Numbers` dashboard." sentence from above to here?

```suggestion
On the :guilabel:`Lots/Serial Numbers` dashboard, products with serial numbers assigned to them are listed by default. Click the
```